bycycle/features/burst.py

```diff
@@ -9,16 +9,13 @@
 ###################################################################################################
 ###################################################################################################
 
-def compute_burst_features(df_shape_features, df_samples, sig,
-                           burst_method='cycles', burst_kwargs=None):
+def compute_burst_features(df_shape_features, sig, burst_method='cycles', burst_kwargs=None):
     """Compute burst features for each cycle.
 
     Parameters
     ----------
     df_shape_features : pandas.DataFrame
```

Comment:
doesn't this input now combine both shape & cyclepoint features? If so, this description seems perhaps slightly out of data. Maybe it should be called `df_features`?